module-4/backend/week_one.md

```diff
@@ -5,15 +5,33 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+* Seeing how some tests were set-up and looking at how to import/require the file into the tests.
 2. What are some tools to help debug JavaScript code?
+* jspry, debugger, console.log, and chrome-dev-tools
 3. What are some tools you need in order to unit test your JavaScript?
+* mocha, chai
 4. What is the syntax for invoking a function?
+* functionName(); or ()
 5. What's `this` in JavaScript?
+* `this` is dependent upon where it is located in the code. If we `console.log(this)` in our dev-tools it will show the window. However, `this` inside a contrcutor object will refer to the object itself like so: 
+```function Dog (name, breed, age) {
+   this.name = name,
+   this.breed = breed,
+   this.age = age,
+}
+```
 6. What is Webpack and why is it useful?
+* Webpack is where the libraries and extensions that we have included in our project can be listed.
```

Comment:
Your answers for webpack and npm are perfect, but they are backwards! 

Webpack - bundles, minifies, etc. 
npm - libraries and packages for the project